### ISSUE TITLE
test input and output

### ISSUE THREAD

**Issue by sgould** [Plested/char-LSTM-RNN#1]
Please include a test input ("bigtxt.txt") and sample output for testing. Moving the input and output filenames to where the hyperparameters are defined may make it easier to configure the code for different experiments.

**Comment by Plested** [comment]
Done. I have included an output file showing test and training loss every 1,000 iterations and final output text from a model with 100 hidden units. This model is too small to be very good, but trains quickly. I have also included output text from a fully trained model with 400 hidden units which is much better, but I need to re-do the training as the figures for text loss were incorrect (I found a bug this morning which I have fixed in the latest version). It will take me a few days to re-do the training for the larger models, but should I have all the output and graphs of iterations vs error for various model complexities and error vs model complexity then.
